.github/workflows/linting.yml

```diff
@@ -24,3 +21,16 @@ jobs:
           clang-format --dry-run -Werror *cpp include/*hpp
           cd -
         done
+
+  cmake-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: clang-format
+      run: |
+        pip install cmakelang
+        cmake-lint $(find . -name CMakeLists.txt) -c .cmake-lint-config.py
+        # NOTE: We ignore C0103, which would enforce that variables are all-caps
+        # NOTE: We ignore C0113, which would enforce that comments are used
+        # NOTE: We set the tab size to 4
+        # NOTE: We set the maximum line length to 100
```

Comment:
Okay, I figured out how to set a config file with these four edits. With these edits, the lint check passes without modification of the code!